hera_sim/beams.py

```diff
@@ -104,21 +388,27 @@ def __eq__(self, other):
 
 
 class PerturbedPolyBeam(PolyBeam):
-    """
-    A PolyBeam in which the shape of the beam has been modified.
+    """A PolyBeam in which the shape of the beam has been modified.
```

Comment:
for documentation cross-referencing, it would be great if you could use 
```
:class:`PolyBeam`
``` 
here instead of just the name.